### DISCUSSION TITLE
Filter by concatenating columns in Supabase

### DISCUSSION THREAD

**Discussion by dule-kitemetric** [supabase/supabase#27471] [Questions] [answered]
I have a table `users` with columns `first_name` and `last_name`. I want to query all users whose name match the a given string for my get user list API with filters.

I can do that with SQL like this:
```
SELECT *
FROM users
WHERE CONCAT(first_name, ' ', last_name) ilike '%string-to-search℅';
```

How to use Supabase to achieve that? I would be very appreciated if anyone can help me.
Current i haven't found a solution for this yet.

**Comment by GaryAustin1** [comment, marked as answer]
You would use a computed field and your filter on that.  It is part of the REST API Supabase uses.  https://postgrest.org/en/stable/references/api/computed_fields.html 

Otherwise using a Postgres generated column is the other way.https://www.postgresql.org/docs/current/ddl-generated-columns.html

**Reply by dule-kitemetric** [reply]
Thank you so much. The solution "using computed fields" works well for me.
For anyone facing this, I can apply filter on the computed field **full_name** as a normal column
`supabaseClient.from('users').select().ilike('full_name', '%string-to-search℅')`
